roles/variables/vars/main.yml

```diff
@@ -1721,7 +1721,7 @@ control_center_properties:
         ksql_http_protocol, ksql_listener_port, control_center_truststore_path, control_center_truststore_storepass,
         control_center_keystore_path, control_center_keystore_storepass, control_center_keystore_keypass) }}"
   oauth:
-    enabled: "{{oauth_enabled}}"
+    enabled: "{{oauth_enabled or schema_registry_oauth_enabled or kafka_connect_oauth_enabled}}"
```

Comment:
why is kafka_rest_oauth_enabled not included here

Reply:
c3 is not a client for kafka rest